### PR TITLE
10825 feature playlist endpoint

### PR DESCRIPTION
**Pull request by Alfro**
BasePlayer playlist endpoint implementation.

### CODE REVIEW

File: src/Pumukit/BasePlayerBundle/Services/SeriesPlaylistService.php [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+namespace Pumukit\BasePlayerBundle\Services;
+
+use Doctrine\ODM\MongoDB\DocumentManager;
+use Pumukit\SchemaBundle\Document\MultimediaObject;
+use Pumukit\SchemaBundle\Document\Track;
+use Pumukit\SchemaBundle\Document\Series;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+
+class SeriesPlaylistService
```
Comment (rubenrua) [inline src/Pumukit/BasePlayerBundle/Services/SeriesPlaylistService.php]: Why a service? You only need a repo

File: src/Pumukit/JWPlayerBundle/Controller/SeriesPlaylistController.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            'object' => $series,
+            'responsive' => true,
+        );
+    }
+
+    /**
+     * @Route("/series_playlist/magic/{secret}", name="pumukit_videoplayer_magicindex", defaults={"show_hide": true, "no_channels": true} )
+     * @Template("PumukitJWPlayerBundle:JWPlayer:player_playlist.html.twig")
+     */
+    public function magicIndexAction(Series $series, Request $request)
```
Comment (rubenrua) [inline src/Pumukit/JWPlayerBundle/Controller/SeriesPlaylistController.php]: Why not only a function with two routes?

File: src/Pumukit/BasePlayerBundle/Services/SeriesPlaylistService.php (hunk 1 above)
Comment (Alfro) [inline src/Pumukit/BasePlayerBundle/Services/SeriesPlaylistService.php]: I'm on it. Sadly, we discussed this after I did the pull request :sweat:

File: src/Pumukit/JWPlayerBundle/Controller/SeriesPlaylistController.php (hunk 2 above)
Comment (Alfro) [inline src/Pumukit/JWPlayerBundle/Controller/SeriesPlaylistController.php]: I'm on it. Sadly, we talked about this on a similar controler after I did this pull request.
I also renamed the controller.

File: src/Pumukit/BasePlayerBundle/Services/SeriesPlaylistService.php (hunk 1 above)
Comment (Alfro) [inline src/Pumukit/BasePlayerBundle/Services/SeriesPlaylistService.php]: But we agreed we would do separate this logic into a service so we could easily add the user filtering later. (The groups logic was done just now) #129

Comment (Alfro) [inline src/Pumukit/BasePlayerBundle/Services/SeriesPlaylistService.php]: Update. Because 7b40b820eeb229b4c96a6ff2fdd2322e1a623089

**Comment by Alfro** [comment]
With #135 accepted, this pull request is already included on the future 2.3.x

**Pull request closed without merging**